### ISSUE TITLE
carousel responsiveness not working in  small sections of Sharepoint site pages

### ISSUE THREAD

**Issue by asadshaikhcodes** [YIZHUANG/react-multi-carousel#232]
I am using react-multi-carousel in one of my Sharepoint webpart(component) project, which shows some Employee profile cards in carousel, as Sharepoint site pages have multiple sections on a page with varying widths, when my component is placed in a small section it messes up my carousel item. I am providing a responsive object with some screen breakpoint, but it does not work on a small section of a page. After so many hours of trying, it seems it works only on specific screen sizes which are mentioned in responsive object but not on specific widths. am I missing something, or what can I do, so the carousel will work on specific widths also. e.g if width is 500px I want to show 1 item, if width is wide enough I need to show multiple items.
kindly refer to below image to see what I am need to achieve. Thanks.
https://user-images.githubusercontent.com/36621106/107761821-009d8080-6d52-11eb-8c11-2539342ff4f0.png

**Comment by Mano534** [comment]
yes i want to show a specific amount of items (slides) on different screen sizes , is there is a way to control that using react multi carosul package
